packages/tldraw/src/lib/utils/export/export.ts

```diff
@@ -19,17 +19,17 @@ export async function getSvgAsImage(
 	svgString: string,
 	options: {
 		type: 'png' | 'jpeg' | 'webp'
-		quality: number
-		scale: number
 		width: number
 		height: number
+		quality?: number
+		pixelRatio?: number
 	}
 ) {
-	const { type, quality, scale, width, height } = options
+	const { type, width, height, quality = 1, pixelRatio = 2 } = options
```

Comment:
should this default to `window.devicePixelRatio` or `editor.getInstanceState().devicePixelRatio`?

Reply:
no, even for 1x projects we want to create it at 2x (144ppi, rather than 72ppi).